autobisect/main.py

```diff
@@ -99,9 +119,18 @@ def _parse_args(argv=None):
             parser.error('Detect mode set to log-limit but no limit set!')
         if args.detect == 'memory' and args.memory is None:
             parser.error('Detect mode set to log-limit but no limit set!')
-
-    if args.branch is None:
-        args.branch = 'central'
+    elif args.target == 'js':
+        if args.detect == 'diff':
+            if args.arg_1 is None or args.args_2 is None:
+                parser.error('Detect mode set to diff but no arguments supplied!')
+        if args.detect == 'hang':
+            if args.hang_time is None:
```

Comment:
Should comparisons to `None` be changed to `if not args.hang_time:`?

Reply:
No because that will also match if hang_time is 0.  I'll add a second condition to check for hang_times <= 0.